src/plugin/settings/engine.cpp

```diff
@@ -525,6 +530,15 @@ void Engine::doGetSettingsOnActiveSettingChange(Result<const IActiveSettingChang
 
     std::map<std::string, std::string> values = toStdMap(shareToPtr(activeSettingChangedAction->settingsValues()));
 
+    // if (!updateModel(&model, &values, {settingId}))
```

Comment:
Do you want this commented out?

Reply:
Woops! There's that commented block, back from the dead! This was from a bad rebase. I'll drop the block (again!).